packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import CashierWalletModalHeader from './cashier-wallet-modal-header';
+import CashierWalletModalBody from './cashier-wallet-modal-body';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '@deriv/stores';
+
+const CashierWalletModal = observer(() => {
+    const store = useStore();
+    const {
+        client: { balance, currency },
+        ui: { is_dark_mode_on, is_cashier_wallet_modal_visible, setIsCashierWalletModalVisible },
+        traders_hub: { is_demo },
+    } = store;
+
+    const tab_content_ref = React.useRef<HTMLDivElement>(null);
+    const [active_tab_index, setActiveTabIndex] = React.useState(0);
+    const [show_wallet_name, setShowWalletName] = React.useState(true);
```

Comment:
```suggestion
    const [show_wallet_name, setShowWalletName] = React.useState<boolean>(true);
```

Reply:
@mahdiyeh-deriv i will apply your comment, but just for my understanding. Why we can't rely on type inference for useState if we use `boolean` as initial value and should strongly type useState hook?